src/components/generics/date-range-picker.tsx

```diff
@@ -0,0 +1,91 @@
+import { Box } from "@mui/material";
+import { DatePicker } from "@mui/x-date-pickers";
+import { DateTime } from "luxon";
+import { useEffect, useState } from "react";
+import { getTimeDifferenceInDays } from "../../utils/time-utils";
+
+/**
+ * Component properties
+ */
+interface Props {
+  dateTimeNow: DateTime;
```

Comment:
Instead of passing dateTimeNow from parent component, could DateTime.now() be directly used instead, since DateTime from luxon is already imported in this component?

Reply:
If I created another instance of dateTimeNow here, it would have a small time difference compared to the one in the parent component. Passing dateTimeNow to children solved this problem